components/director/internal/domain/systemauth/converter.go

```diff
@@ -45,29 +48,41 @@ func (c *converter) ToGraphQL(in *model.SystemAuth) (graphql.SystemAuth, error)
 		return nil, err
 	}
 
+	systemAuthTypeApplication := graphql.SystemAuthReferenceTypeApplication
```

Comment:
In the return clause I guess as value of the desired field? If that's the case, no, because we need a pointer and we cannot use `&graphql.SystemAuthReferenceTypeApplication` as value of `Type`